ModsenOnlineStore.Store.API/AutoMapperProfile.cs

```diff
@@ -0,0 +1,16 @@
+﻿using AutoMapper;
+using ModsenOnlineStore.Store.Domain.DTOs.ProductDTOs;
+using ModsenOnlineStore.Store.Domain.Entities;
+
+namespace ModsenOnlineStore.Store.API
+{
+    public class AutoMapperProfile : Profile
+    {
+        public AutoMapperProfile()
```

Comment:
Name it ProductProfile as it should store only product related mapping configurations

Reply:
fixed